lib/logstash/filters/uuid.rb

```diff
@@ -45,13 +45,12 @@ def filter(event)
     return unless filter?(event)
 
     if overwrite
-      event[target] = SecureRandom.uuid
+      event[target] = SecureRandom.uuid.force_encoding("UTF-8")
```

Comment:
It isnt needed, it was some OSX bug, I created a vagrant box for testing and utf8 wasnt an issue. I'm going to remove it.